crates/loader/src/local/assets.rs

```diff
@@ -260,3 +290,35 @@ fn is_absolute_guest_path(path: impl AsRef<Path>) -> bool {
     // paths specifically using Unix logic rather than the system function.
     path.as_ref().to_string_lossy().starts_with('/')
 }
+
+/// Convert strings to glob patterns
+fn convert_strings_to_glob_patterns<T: AsRef<str>>(files: &[T]) -> Result<Vec<glob::Pattern>> {
```

Comment:
If you pass the base source directory in here, you can make the glob patterns absolute, and then shouldn't need to keep the relative source around in the FileMount.

Reply:
Yeah, added `rel` here. Thank you.